console/backend/pkg/model/notebook.go

```diff
@@ -0,0 +1,97 @@
+package model
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+type NotebookResource struct {
+	TotalCPU    int64 `json:"totalCPU"`
+	TotalMemory int64 `json:"totalMemory"`
+	TotalGPU    int64 `json:"totalGPU"`
+}
+
+// NotebookInfo for the http request
+type NotebookInfo struct {
+	// Id: unique id
+	Id string `json:"id"`
+	// Name: Notebook name
+	Name string `json:"name"`
+
+	// status of Notebook
+	NotebookStatus string `json:"notebookStatus"`
+
+	// url to the notebook
+	Url string `json:"url"`
+
+	// Namespace
+	Namespace string `json:"namespace"`
+	//
```

Comment:
remove `//` or complete the comments

Reply:
removed, the filed name is self-explanatory enough